korlibs-datastructure/test/korlibs/datastructure/ByteArrayDequeTest.kt

```diff
@@ -26,7 +25,9 @@ class ByteArrayDequeTest {
         data.write(byteArrayOf(0x99.toByte(), 0x88.toByte(), 0x33, 0x44, 0x55, 0x66, 0xAA.toByte()))
         assertEquals(8, data.availableRead)
         assertEquals(0, data.availableWriteWithoutAllocating)
-        assertEquals(listOf(0x7F, 0x99, 0x88, 0x33, 0x44, 0x55, 0x66, 0xAA), data.readBytesUpTo(8).map { it.toInt() and 0xFF })
+        assertEquals(
+            listOf(0x7F, 0x99, 0x88, 0x33, 0x44, 0x55, 0x66, 0xAA),
+            data.readBytesUpTo(8).map { it.toInt() and 0xFF })
```

Comment:
Why this change? Can we have moved files without changing imports or reformatting? We can do reformatting / optimizing imports in a separate PR if required.

Reply:
I always end the PR by formatting. Turning back now is too much effort. If you prefer other formatting than I suggest edding automated formatting so it happens when someone pushes the commit.